pkg/apis/core/validation/validation.go

```diff
@@ -3165,6 +3167,17 @@ func validateContainerCommon(ctr *core.Container, volumes map[string]core.Volume
 		allErrs = append(allErrs, field.Required(path.Child("image"), ""))
 	}
 
+	// Adding API validation for images using the dockerref vendored package
```

Comment:
This comment should be updated as there is no direct call of dockerref API in this file.

Reply:
Updated.